easybuild/easyconfigs/h/HepMC/HepMC-2.06.11-gompi-2022a.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>
+
+easyblock = 'CMakeMake'
+
+name = 'HepMC'
```

Comment:
Since we already have `HepMC3` (https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/h/HepMC3), should we perhaps name this `HepMC2`?

Reply:
Hi, I picked the name since the headerfiles are named in that way:
```
/usr/include $ ls | grep Hep
HepMC
HepMC3
```
But it is a bit messy a seen here: https://repology.org/project/hepmc/versions
* Spack, Debian/Ubuntu, Fedora and Nix have HepMC and HepMC3 
* OpenSUSE has HepMC2 and HepMC
* Arch has HepMC, HepMC2 and HepMC3

I'd go with the majority.